testsuite/features/step_definitions/command_steps.rb

```diff
@@ -71,6 +71,12 @@
   $server.run("curl --insecure --proxy '#{proxy}' --proxy-anyauth --location '#{url}' --output /dev/null")
 end
 
+Then(/^it should be possible to use the FTP server$/) do
+  # TODO: use a URL in Provo after the release
+  url = 'ftp://minima-mirror.mgr.suse.de:445/rhn/manager/download/test-channel-x86_64/repodata/repomd.xml'
```

Comment:
That domain should be parameterizable somehow... depending if you are in NUE or PRV.

Reply:
yes, but then I have to do the same for the SLS pillar file, and that complicates another step

I'd like to do something simple today, we can make it more intelligent some other day

Reply:
I'll create a card to make it better